v3/src/components/graph/models/graph-content-model.ts

```diff
@@ -351,15 +350,108 @@ export const GraphContentModel = DataDisplayContentModel
         : casePrimaryValue
       const firstCount = legendAttrID ? totalCases : casesInSubPlot
       const secondCount = legendAttrID ? casesInSubPlot : totalCases
-
+      const percent = float(firstCount / secondCount * 100)
       // <n> of <m> <category> (<p>%) are <legend category>
       const attrArray = [
         firstCount, secondCount, caseCategoryString, percent, caseLegendCategoryString
       ]
-
       return t("DG.BarChartModel.cellTipPlural", {vars: attrArray})
     }
   }))
+  .views(self => ({
+    cellParams(primaryCellWidth: number, primaryHeight: number) {
```

Comment:
In an earlier comment, the question was asked about where methods like `cellParams` and `mapOfIndicesByCase` should live. Perhaps they would make sense in the `SubPlotCells` class? Not a requirement for this PR, but something to consider.

Reply:
I tried moving those but ran into some confusing problems. It's not clear that they'd be easy to work out, so I'm going to opt out of that for this PR.